### ISSUE TITLE
Python installed two times with different versions

### ISSUE THREAD

**Issue by lucaGazzola** [nikolaik/docker-python-nodejs#85]
The image building process at the moment (at least for some images) installs python two times with different versions (see https://github.com/cheshire-cat-ai/core/pull/290#issuecomment-1586337963). This is not ideal, in particular for the `slim` images.

**Comment by nikolaik** [comment]
I'll have a look at why nodesource debs want to install their own python, could it be bacause of node-gyp or something in that area? I'll consider switching the install method to something similar to official node images instead. Are the alpine images an alternative for now?

**Comment by zioproto** [comment]
It is easy to answer why nodesource debs want to install their own python:

```
$ wget https://deb.nodesource.com/node_18.x/pool/main/n/nodejs/nodejs_18.16.0-deb-1nodesource1_arm64.deb
$ dpkg -I nodejs_18.16.0-deb-1nodesource1_arm64.deb  | grep Depends
 Depends: libc6 (>= 2.28), libgcc1 (>= 1:4.2), libstdc++6 (>= 5.2), python3-minimal, ca-certificates
```
There is a dependency on `python3-minimal`

In Debian bullseye the python version is 3.9
Could you please check if Debian bullseye  offers python 3.10 as deb packages? I believe it does not but I could be wrong.

I would check if you can change the default python version before pulling the packages with:
```
update-alternatives --list python3-minimal
```
but it is not working for me.

You could easily solve this problem using a more recent Debian version

Please let me know if this helps.

**Comment by nikolaik** [comment]
This should be fixed with #109 which starts using the oficially distributed node.js releases instead of nodesource deb packages.
